### ISSUE TITLE
Setting S3 as back store for remote server

### ISSUE THREAD

**Issue by snarasimhan1997** [evidentlyai/evidently#1021]
Hey, I was wondering if its possible to configure a remote evidently server to use say an s3 bucket for all its metadata and snapshot storage. In [this link](https://docs.evidentlyai.com/user-guide/monitoring/workspace_project#remote-snapshot-storage), it mentioned that the UI server can read snapshots from a remote FS like AWS but it does not mention if we can also write to the FS.

My final solution would look like running the UI server in an EC2 instance connected to an EBS volume. I want my UI server to use this EBS storage (or s3 too) to read and write snapshots. Is this possible?

**Comment by mike0sv** [comment]
Yes it can. If you setup s3 as a storage backend, UI server will upload new snapshots there as well
